commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/MeanTest.java

```diff
@@ -310,13 +308,4 @@ static Stream<Arguments> testCombineNonFinite() {
             Arguments.of(new double[][] {{Double.NEGATIVE_INFINITY, -Double.MAX_VALUE, -Double.MIN_VALUE}, {Double.MAX_VALUE, Double.MIN_VALUE}}, Double.NEGATIVE_INFINITY)
         );
     }
-
-    // Helper function to compute the expected value of Mean using BigDecimal.
-    private static double computeExpected(double[] values) {
```

Comment:
You could leave this function here and have the ported function return the BigDecimal result. This then becomes:
```java
private static double computeExpected(double[] values) {
    return TestHelper.computedExpectedMean(values).doubleValue();
}
```